internal/command/volumes/extend.go

```diff
@@ -72,7 +74,18 @@ func runExtend(ctx context.Context) error {
 		return err
 	}
 
-	sizeGB := flag.GetInt(ctx, "size")
+	sizeFlag := flag.GetString(ctx, "size")
+	sizeGB := 0
+	if sizeFlag[0] == '+' {
+		volume, err := flapsClient.GetVolume(ctx, volID)
```

Comment:
To make +1 work, I had to get the volume to determine its current size, which makes it slower.
But I do this only if there's a + as the first character. Otherwise, I keep treating it as a replacement value for current one.